### PR TITLE
Fix for 'Impossible Traveler Response' XSIAM playbook

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10957

## Description
The 'EndpointID' playbook input was not configured for the 'Containment Plan', which is critical for the execution of the 'core-get-endpoints' automation within the 'Get endpoint info' task.

### CODE REVIEW

File: Packs/Core/ReleaseNotes/3_0_44.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Impossible Traveler Response
+
+- Updated the *'EndpointID'* playbook input configuration within the **'Containment Plan'** sub-playbook.
```
Comment (ssokolovich) [inline Packs/Core/ReleaseNotes/3_0_44.md:6]:
```suggestion
- Updated the *'EndpointID'* playbook input value within the **'Containment Plan'** sub-playbook.
```

File: Packs/Core/Playbooks/playbook-Impossible_Traveler_README.md [hunk 2]
```diff
@@ -122,4 +122,4 @@ This playbook does not use any integrations.
 
 ---
 
-![Impossible Traveler Response](../doc_files/Impossible_Traveler.png)
+![Impossible Traveler Response](../doc_files/Impossible_Traveler_Response.png)
```
Comment (ssokolovich) [inline Packs/Core/Playbooks/playbook-Impossible_Traveler_README.md:125]:
```suggestion
![Impossible Traveler Response](../doc_files/Impossible_Traveler.png)
```

File: Packs/Core/ReleaseNotes/3_0_44.md (hunk 1 above)
Comment (TalNos) [reply Packs/Core/ReleaseNotes/3_0_44.md:6]: done

File: Packs/Core/Playbooks/playbook-Impossible_Traveler_README.md (hunk 2 above)
Comment (TalNos) [reply Packs/Core/Playbooks/playbook-Impossible_Traveler_README.md:125]: done

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**